lte/gateway/c/core/oai/tasks/nas/ies/ApnAggregateMaximumBitRate.c

```diff
@@ -113,52 +118,120 @@ void bit_rate_value_to_eps_qos(
     ApnAggregateMaximumBitRate* apn_ambr, uint64_t ambr_dl, uint64_t ambr_ul) {
   uint64_t ambr_dl_kbps = ambr_dl / 1000;  // ambr_dl is expected in bps
   uint64_t ambr_ul_kbps = ambr_ul / 1000;  // ambr_ul is expected in bps
```

Comment:
should not we now also using units explicitly in the function call?

Reply:
Updated